loader/trampoline.c

```diff
@@ -729,11 +729,15 @@ LOADER_EXPORT VKAPI_ATTR void VKAPI_CALL vkDestroyInstance(VkInstance instance,
     }
 
     if (messenger_setup) {
+        loader_log(ptr_instance, VULKAN_LOADER_INFO_BIT, 0, "vkDestroyInstance: destroying all debug util messengers");
+
```

Comment:
This implies "all", even those not attached to this instance. Could we somehow tell the user which instance is having all the messengers destroyed?